system-tests/ft-suite/src/test/kotlin/com/hotels/styx/logging/HttpMessageLoggingSpec.kt

```diff
@@ -0,0 +1,118 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.logging
+
+import ch.qos.logback.classic.Level.INFO
+import com.hotels.styx.StyxConfig
+import com.hotels.styx.StyxServer
+import com.hotels.styx.api.HttpHeaderNames
+import com.hotels.styx.api.HttpRequest
+import com.hotels.styx.client.StyxHttpClient
+import com.hotels.styx.startup.StyxServerComponents
+import com.hotels.styx.support.matchers.LoggingTestSupport
+import io.kotlintest.Spec
+import io.kotlintest.specs.FeatureSpec
+import reactor.core.publisher.toMono
+
+class HttpMessageLoggingSpec : FeatureSpec() {
+
+    init {
+        feature("Styx request/response logging") {
+
+            scenario("Logger should hide cookies and headers") {
+                val proxyHost = "${styxServer.proxyHttpAddress().hostName}:${styxServer.proxyHttpAddress().port}"
```

Comment:
Use `StyxServer.proxyHttpHostHeader ` extension method from support/styxServerProvider.kt to simplify `proxyHost`.